Extension/package.json

```diff
@@ -93,6 +93,430 @@
           "description": "%c_cpp.configuration.clang_format_style.description%",
           "scope": "machine-overridable"
         },
+        "C_Cpp.formatting": {
+          "type": "string",
+          "enum": [
+            "clangFormat",
```

Comment:
The "Default" enum was removed -- is that breaking change going to be an issue? What happens for existing users that have "Default" set?

Reply:
Default was supposed to stay. It has been added back in the latest push.